src/cli/cli.cpp

```diff
@@ -4612,12 +4612,9 @@ otError Interpreter::ProcessDiag(uint8_t aArgsLength, Arg aArgs[])
 {
     otError error;
     char *  args[kMaxArgs];
-    char    output[OPENTHREAD_CONFIG_DIAG_OUTPUT_BUFFER_SIZE];
+    char    output[OPENTHREAD_CONFIG_DIAG_OUTPUT_BUFFER_SIZE] = {'\0'};
```

Comment:
I think you intend to have all chars set to zero, is this right?
If so, how about `memset()` the `output` to zero instead.

Reply:
SGTM, Thanks.